doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -448,11 +448,97 @@ JCL      - a786f96b13 based on jdk-11+21)
 
 ----------------------------------
 
-## MacOS
+## macOS
 :apple:
+The following instructions guide you through the process of building a macOS **OpenJDK V11** binary that contains Eclipse OpenJ9. This process can be used to build binaries for macOS 10.
 
-:construction:
-We haven't created a full build process for macOS yet? Watch this space!
+### 1. Prepare your system
+:apple:
+You must install a number of software dependencies to create a suitable build environment on your system:
+
+- [Xcode 9.4](https://xcodereleases.com/).
```

Comment:
Should we be suggesting downloading from an unofficial website, when an official one exists? https://developer.apple.com/download/more/ (requires an account)

Reply:
Ah, certainly not. It was the first one I found. Altering now.